vms/platformvm/state/diff_test.go

```diff
@@ -80,8 +80,7 @@ func TestDiffCurrentValidator(t *testing.T) {
 
 	lastAcceptedID := ids.GenerateTestID()
 	state := NewMockState(ctrl)
-	// Called in NewDiff
-	state.EXPECT().GetTimestamp().Return(time.Now()).Times(1)
+	state.EXPECT().GetTimestamp().Return(time.Now()) // Called in NewDiff
```

Comment:
not strictly related to this diff, but I just find previous line arrangement very annoying. We could drop the comment altogether as well.